wordpress-mobile/WordPress-Android
# Add floating "New notes" button

Fixes #4743 by adding a floating "New notes" button that appears when a new notification is received while looking at the notifications list (note: it doesn't appear if you're already at the top of the list).
When you tap on it you're taken all the way up without having to scroll the list.

Also merged develop into this branch. 

To test: 
- Scroll the notifications list.
- Make a comment on a blog.
- Wait till the PN reaches the device.
- You should see the button appearing at the bottom of the list.
- Tapping the button should take you to the top of the list.

cc @kwonye
